reference/reference-styling.html

```diff
@@ -235,12 +239,13 @@ <h3 id="custom-strokes">Custom Strokes</h3>
     </g>
 </svg>
         </div>
-        <p>This SVG is composed of six superimposed elements, and the appropriate components will be referenced with the SVG <code>use</code> element. Most of these elements are single <code>path</code> elements in the example above, but can be groups of elements so long as a parent <code>g</code> element has the appropriate <code>id</code> attribute.</p>
+        <p>This SVG is composed of seven superimposed elements, and the appropriate components will be referenced with the SVG <code>use</code> element. Most of these elements are single <code>path</code> elements in the example above, but can be groups of elements so long as a parent <code>g</code> element has the appropriate <code>id</code> attribute.</p>
         <ul>
             <li><code>spoke-n</code>: a spoke pointing north (which will be rotated for east, south, and west).</li>
             <li><code>spoke-ne</code>: a spoke pointing northeast (which will be rotated for southeast, southwest, and northwest).</li>
             <li><code>x-spoke-n</code>: a "no" spoke pointing north (which will be rotated for east, south, and west).</li>
             <li><code>x-spoke-ne</code>: a "no" spoke pointing northeast (which will be rotated for southeast, southwest, and northwest).</li>
+            <li><code>double-spoke-n</code>: a parallel set of spokes pointing north (which will be rotated for east, south, and west).</li>
```

Comment:
If expanded to multiple levels, I propose element names like spoke-n-2, spoke-ne-2, etc. Again, it's OK if the default SVG does not supply a full set.